service/kbupd/build.rs

```diff
@@ -17,6 +17,26 @@ fn main() {
 
     let mut protoc = prost_build::Config::new();
     protoc
+        .type_attribute(".", "#[derive(serde::Serialize)]")
+        .field_attribute("node_id", "#[serde(with = \"serde_with::As::<serde_with::hex::Hex>\")]")
+        .field_attribute(
+            "EnclaveFrontendPartitionStatus.group_id",
+            "#[serde(with = \"serde_with::As::<serde_with::hex::Hex>\")]",
+        )
+        .field_attribute(
+            "EnclaveFrontendRangeStatus.group_id",
+            "#[serde(with = \"serde_with::As::<serde_with::hex::Hex>\")]",
+        )
+        .field_attribute(
+            "EnclaveReplicaPartitionStatus.group_id",
+            "#[serde(with = \"serde_with::As::<serde_with::hex::Hex>\")]",
+        )
+        .field_attribute(
+            "EnclaveReplicaPartitionStatus.service_id",
+            "#[serde(with = \"serde_with::As::<Option<serde_with::hex::Hex>>\")]",
```

Comment:
IIUC this would become `with = "kbupd_util::hex"`
```suggestion
            "#[serde(with = \"kbupd_util::hex\")]",
```